wp-content/themes/workingnyc/views/search/search-results.twig

```diff
@@ -20,65 +20,67 @@
       </header>
   </div>
 
-  <section class="px-4 tablet:px-6">
-    {% if term != '' %}
-      {% if posts|length > 0 %}
-        {% block list_sidebar %}
-        {% endblock %}
-        <div class="tablet:px-6">
-          {% block before_list %}
-          <h2 class="text-p font-p inline-block mt-4 mb-2">
-            {% if posts|length > 1 %}
-              {{ __('Showing {{ TOTAL_VISIBLE }} results for "{{ TERM }}".', 'WNYC')|replace({'{{ TOTAL_VISIBLE }}': posts|length, '{{ TERM }}': term}) }}
-            {% else %}
-              {{ __('Showing 1 result for "{{ TERM }}".', 'WNYC')|replace({'{{ TERM }}': term}) }}
-            {% endif %}
-          </h2>
-          <div class="mb-4">
-            <button class="btn-tag btn-primary" id='all-button'>{{__('All', 'WNYC')}}</button>
-            <button class="btn-tag" id='jobs-button'>{{__('Jobs', 'WNYC')}}</button>
-            <button class="btn-tag" id='programs-button'>{{__('Programs', 'WNYC')}}</button>
-          </div>
+  <div data-js="search-results-list">
```

Comment:
The only non-whitespace change in this file is adding this line. It allows search.js to attach to this HTML element and replace it with the appropriate HTML/JavaScript it renders